dashboard/app/views/devise/registrations/new.html.haml

```diff
@@ -155,3 +157,8 @@
       });
     }
   });
+
+  $(document).ready(function() {
+    var schoolInfoElement = document.getElementById('schooldropdown-block');
+    ReactDOM.render(React.createElement(window.dashboard.schoolInfoBox), schoolInfoElement);
+  });
```

Comment:
If you use the script include approach from my previous comment, this can then live in your JS.